scripts/quests/windurst/Let_Sleeping_Dogs_Lie.lua

```diff
@@ -56,15 +61,20 @@ quest.sections =
             ['Chomoro-Kyotoro'] = quest:event(489),
             ['Paku-Nakku'] =
             {
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.items.BUNCH_OF_BLAZING_PEPPERS) and quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(494)
+                    end
+                end,
+
                 onTrigger = function(player, npc)
-                    if quest:getVar(player, 'Prog') == 1 then
-                        return quest:progressEvent(494, 653393087, 1, 0, 1200, 8388608, 3749, 4095, 2) -- 0, 0, 0, 0)
-                    elseif quest:getVar(player, 'Prog') == 3 then
+                    if quest:getVar(player, 'Prog') == 3 then
                         return quest:progressEvent(499)
                     else
                         return quest:event(482)
                     end
                 end,
+
             },
             ['Pechiru-Mashiru'] =
             {
```

Comment:
can't comment on it directly, but below can you make return quest:event(497) -> return quest:progressEvent(497)

Reply:
updated.